app/src/main/java/com/parishod/watomatic/activity/about/AboutActivity.java

```diff
@@ -0,0 +1,83 @@
+package com.parishod.watomatic.activity.about;
+
```

Comment:
Wish this was kept in Kotlin though. We are trying to migrate from Java to Kotlin bit by bit but its okay for now :)